patch/MultiDelay/MultiDelay.cpp

```diff
@@ -0,0 +1,138 @@
+#include "daisysp.h"
+#include "daisy_patch.h"
+#include <string>
+
+#define MAX_DELAY static_cast<size_t>(48000 * 1.f)
+
+using namespace daisy;
+using namespace daisysp;
+
+DaisyPatch patch;
+
+DelayLine <float, MAX_DELAY> DSY_SDRAM_BSS delMems[3]; 
+
+struct delay
+{
+    DelayLine <float, MAX_DELAY>  *del;
+    float currentDelay;
+    float delayTarget;
+};
+
+delay delays[3];
+Parameter params[4];
+
+float feedback;
+int drywet;
+
+void ProcessControls();
+
+float UpdateDelay(delay &delay, float in)
+{
+    //set delay times
+    fonepole(delay.currentDelay, delay.delayTarget, .004f);
+    delay.del->SetDelay(delay.currentDelay);
+    
+    float read = delay.del->Read();
+    delay.del->Write((feedback * read) + (1 - feedback) * in);
```

Comment:
The feedback was a bit phase-y when high. I changed this line to:

```cpp
delay.del->Write((feedback * read) + in);
```

and felt the higher feedback settings become a bit more expressive.